src/PowerWAF.cpp

```diff
@@ -24,38 +24,75 @@
 using namespace ddwaf;
 using namespace std::literals;
 
-PowerWAF::PowerWAF(PWManifest&& manifest_, rule_vector&& rules_,
-                   flow_map&& flows_, const ddwaf_config* config)
-    : manifest(std::move(manifest_)),
-      rules(std::move(rules_)),
-      flows(std::move(flows_))
+namespace {
+obfuscator obfuscator_from_config(const ddwaf_config* config)
 {
-    if (config != nullptr)
-    {
-        if (config->maxArrayLength != 0)
+    std::string_view key_regex, value_regex;
+
+    if (config != nullptr) {
```

Comment:
Not enabling it by default is okay to me if you think everyone will be able to upgrade to this version easily.

Reply:
We can't really do anything else if we want to provide flexibility, given that there are no defaults in C structs.